p2p/ipld/read_test.go

```diff
@@ -168,7 +163,7 @@ func TestRetrieveBlockData(t *testing.T) {
 			colRoots := rootsToDigests(rawColRoots)
 
 			// limit with deadline retrieval specifically
-			ctx, cancel := context.WithTimeout(ctx, time.Second*2)
+			ctx, cancel := context.WithTimeout(ctx, time.Minute*1)
```

Comment:
I think this might just have been an issue with my crappy mac. Running the test normally works fine but the -race flag slows it down to about 45 seconds on my 2 cpu core mac. I'll try it out on my PC and see how I go, probably a better machine for multithreading applications.

Reply:
512 is just the number of iterations of that loop that can run simultaneously but if dag.Get is spinning up a high number of go routines the total could be significantly higher.